book/proposal/outline.md

```diff
@@ -0,0 +1,125 @@
+# Outline for Design Thinking for Developers
+
+## Chapter 1: Organic Software with Design Thinking and Domain Driven Development
+
+Organic software is the modern ideal and quality needed if we seek to perpetuate the success and social responsibility of open source to a wider set of problems. We seek to serve the whole of life, seeking solutions which effortlessly enhance our environments in an informed, thoughtful and responsible manner.
```

Comment:
I think this persuasive, philosophical description of "organic software" is better placed after the itemized, concrete definition below (otherwise you may lose the reader, who may be thinking "What is this 'organic software' that's the modern ideal? How have I not heard of it before?")